Fixtures/Miscellaneous/DifferentProductTargetName/Package.swift

```diff
@@ -0,0 +1,12 @@
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+    name: "Foo",
+    products: [
+        .executable(name: "Foo", targets: ["Foo"]),
```

Comment:
Name of the product and the target are the same, why is the fixture called `DifferentProductTargetName` then?

Reply:
Good catch